doc/classes/@GDScript.xml

```diff
@@ -304,6 +320,10 @@
 			</argument>
 			<description>
 				Returns [b]e[/b] raised to the power of 's'. [b]e[/b] sometimes called "Euler's number" is a mathematical constant whose value is approximately 2.71828.
```

Comment:
If you amend the commit, maybe:
`Raises the Euler's constant [b]e[/b] to the power of 's' and returns it. [b]e[/b] has an approximate value of 2.71828.`